### DISCUSSION TITLE
cv91 variations

### DISCUSSION THREAD

**Discussion by TheKnightWho** [psb1558/Junicode-font#115] [Ideas]
Hi - a couple of requests:

1. Would it please be possible to enable the cv91 variations for the combining tilde overlay as well? It would be nice to be able to adjust the tilde when combining with a 'b' or 'd', for example.

2. Could the combining overlay stroke anchor for the letter 'l' please be adjusted slightly upwards? Currently, it can become difficult to make out at smaller sizes as it usually lies flush with the tops of the minuscules either side. For example: 
![image](https://user-images.githubusercontent.com/6895210/156483438-1438939f-b48c-45e8-afbe-ab1c8c5fefc9.png)
I appreciate there needs to be a distinction between that and 'l' with a high stroke, but a slight adjustment upwards would help a lot.

**Comment by psb1558** [comment]
1. Not a problem.

2. I see  the problem, but I'll need to study the best way to handle it, since simply moving the anchor upwards might affect other combinations. BTW another way to avoid unsightly collision with surrounding letters is to use U+019A l with bar instead of the combo l + U+0335:

<img width="106" alt="image" src="https://user-images.githubusercontent.com/16215767/156500715-f7ff11d2-396a-456a-abfe-ac648a04cdeb.png">

**Reply by TheKnightWho** [reply]
Great - thanks. I'm trying to avoid using precomposed characters wherever possible, as U+019A isn't recognised as an 'l' for the purposes of text search. It's an awkward balance.

**Comment by psb1558** [comment]
Understood. Looking over the situation today, I thought the bar was indeed sitting a little low, and it wouldn't hurt anything to raise the anchor.

**Reply by psb1558** [reply]
I have also added (or rather made accessible) a narrow version of the short stroke (U+0335), via `cv91[2]`.

**Comment by psb1558** [comment]
What I'm thinking of doing (and working on) is to eliminate cv91 altogether and make it all contextual:

- A short version of U+0334 and U+0335 will always be selected for l i j.
- A wide version of these same overlay marks will always be selected for b, d, etc. and nudged left or right as appropriate.
- Contextual kerning will be applied as needed.

Examples:
<img width="686" alt="image" src="https://user-images.githubusercontent.com/16215767/156858265-88b4a305-07a9-43cf-99f8-31968c7950a1.png">
There seems little reason to burden users with decisions that could just as well be automatic.

**Reply by TheKnightWho** [reply]
Thanks for this. I've just been experimenting with this, and it seems to work well. Definitely makes my life easier, and I think you could always leave cv91 in place as an option if someone has a particular need to adjust them.

I've noticed that U+334 isn't adjusting automatically for 'j', but I have no idea if that combination is ever used anyway:
![image](https://user-images.githubusercontent.com/6895210/157092330-ea2c31a8-a398-46c0-8330-4d39bc31f5f2.png)

**Comment by psb1558** [comment]
I don't know if anyone needs j with overlay tilde either, but it increases the size of the font by approximately 2 bytes to anticipate the possibility.

I'm always on the lookout for a way to free up a cvNN feature, since they're almost all used, so . . . no cv91.
